ocean_lib/ocean/ocean_assets.py

```diff
@@ -248,16 +258,73 @@ def _default_metadata(self, name: str, tx_dict: dict, type="dataset") -> dict:
         return metadata
 
     @enforce_types
-    def _create_1dt(self, metadata, files, tx_dict, wait_for_aqua):
-        """Call create(), focusing on just one datatoken"""
+    def _create_1dt(
```

Comment:
Is a possible way to find another name for this bundle function? Because `_create_1dt` focuses on just one datatoken, but here we included the data NFT, the datatoken & the data asset

Reply:
this is still an internal function. In the next iteration, when I add the possibility to add pricing schema, I will rename it something smart, like `create_with_bundle` or something similar. This is Trent's naming for now and I chose to modify just the contents until I change the "effect" as well. Do you think that could work for now?